website/static/js/file-browser.js

```diff
@@ -0,0 +1,284 @@
+/**
+ * Builds full page project browser
+ */
+'use strict';
+
+var Treebeard = require('treebeard');   // Uses treebeard, installed as plugin
+var $ = require('jquery');  // jQuery
+var m = require('mithril'); // exposes mithril methods, useful for redraw etc.
+var ProjectOrganizer = require('js/project-organizer');
+var $osf = require('js/osfHelpers');
+
+/**
+ *  Options for fileBrowser
+ */
+var defaults = {
+    wrapper : '#fileBrowser',  // Default ID for wrapping empty div, all contents will be filled in.
+    tboptions : {},
+    fullWidth : true
+};
+
+/**
+ * Initialize File Browser. Prepeares an option object within FileBrowser
+ * @constructor
+ */
+var FileBrowser = {
+    controller : function (args) {
+        var self = this;
+        self.data = [];
+        self.isLoadedUrl = false;
+
+        // View States
+        self.showInfo = m.prop(false);
+
+        self.collections = [
+            { id:1, type : 'collection', label : 'All My Projects', path : 'users/me/nodes/', pathOptions : {  query : { 'filter[registration]' : 'false'} } },
+            { id:2, type : 'collection', label : 'All My Registrations', path : 'users/me/nodes/', pathOptions : { query : {  'filter[registration]' : 'true'} } },
+            { id:3, type : 'collection', label : 'Nodes', path : 'users/me/nodes/', pathOptions : {}}
+        ];
+        self.filesData = m.prop($osf.apiV2Url(self.collections[0].path, self.collections[0].pathOptions));
+
+        self.breadcrumbs = m.prop([
+            { label : 'All My Projects', url : 'http://localhost:8000/v2/users/me/nodes/?filter%5Bregistration%5D=false', type : 'collection'}
```

Comment:
Please use `encodeURIComponent` or some other helper method for encoding query string params.